recipes/p-sea/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: p-sea
+  version: "0.1" 
+
+source:
+  fn: P-SEA.tar.gz
+  url: ftp://ftp.lmcp.jussieu.fr/pub/sincris/software/protein/p-sea/P-SEA.tar.gz
+  md5: 405211399405ebd8784bdac329b42697
+
+build:
+  number: 0
+  skip: True # [osx]
+
+requirements:
+  build:
+    - gcc
+  run:
+
+test:
+  commands:
+    - p-sea -h
+
+about:
+  summary:  new efficient assignment of secondary structure from C_alpha.
```

Comment:
I think using "New" here is hardly true anymore, how about just "Efficient ..."?